ani-cli

```diff
@@ -335,7 +335,27 @@ history)
 esac
 
 # playback & loop
-play_episode
+play() {
```

Comment:
play is an existing program (usually in the sox package). Not that it matrers to us, but it's not that elegant. I'm open if we want to change it or not

Reply:
how about `parse_episode_range` ? imo it would make more sense tbh